File: Loiane_Training_Java/Java_Basico/Exer Aula 18_19/Ex17.java
public class Ex17 {
    public static void main(String[] args) {
        int[] a = { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 40 };

        var cont = 0;
        for (int i = 0; i < a.length; i++) {
            if (a[i] > 35) {
                cont++;
            }
        }

        System.out.println("Quantidade de pessoas acima de 35 anos: " + cont);
    }
}

File: loiane-training-csharp/procedural-poo/e/Ex17.cs
using System;

class Ex17
{
    static void Main(string[] args)
    {
        int[] vetorA = new int[10];
        int contador = 0;
        for (int i = 0; i < vetorA.GetLength(0); i++)
        {
            int idade = int.Parse(Console.ReadLine());
            vetorA[i] = idade;

            if (idade > 35)
            {
                contador++;
            }
        }

        Console.WriteLine("Quantidade de pessoas com mais de 35 anos: {0}", contador);
    }
}